### ISSUE TITLE
Incorrect syntax for OpenID Connect trust policy

### ISSUE THREAD

**Issue by mbogh** [github/docs#18727]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services

### What part(s) of the article would you like to see updated?

I want to update the code snippet in section [Configuring the role and trust policy](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#configuring-the-role-and-trust-policy) to match the syntax provided by AWS in [Using multiple keys and values](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_multi-value-conditions.html#reference_policies_multi-key-or-value-conditions).

Basically add `ForAllValues:` in front of `StringEquals`.

**Comment by mbogh** [comment]
Glitch in the matrix, Trust Policy editor is no longer complaining about it.
